bots/service.go

```diff
@@ -152,18 +182,19 @@ func (s *Service) Create(botID string, botVersion int, name string, params map[s
 		return
 	}
 
-	store := &BotKVStore{
-		botID,
-		botVersion,
+	store := Datastore{
+		ds.NewKey(botID),
 		s.node,
 	}
+	botStore := nsds.Wrap(store, ds.NewKey(botID))
```

Comment:
Using the Datastore method for namespacing bot's key value stores. so a bot's key will automatically be ns'd beneath the bot's id (as a key)